rules/windows/collection_favicon_masquerading.toml

```
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2023/02/16"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/02/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of a suspicious file with the ICO extension. Attackers may masquerade documents, sensitive
+information, and compressed files as files that can be served on webservers or uploaded to web services for exfiltration
+without raising suspicions.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential ICO File Masquerading"
+risk_score = 47
+rule_id = "6c04dff7-0c5a-42b4-8f50-bf5590a51605"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Collection"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where event.action != "deletion" and
+/*
+  Magic numbers / File Signatures
+  000001 -> ICO / 89504E -> PNG
+  FFD8FF -> JFIF, JPE, JPEG, JPG
+  474946 -> GIF / 424D38 -> BMP
+  424D36 -> DIB
+*/
+  file.extension == "ico" and file.size > 700 and not
```

Feels safer to be case insensitive. (IIRC, extensions are not normalized by the endpoint agent)
```suggestion
  file.extension : "ico" and file.size > 700 and not
```